apps/src/lab2/views/components/guide/Guide.module.scss

```diff
@@ -0,0 +1,19 @@
+.guide {
+  gap: 20px;
+  width: 40%;
+  top: 60px;
+  right: 80px;
```

Comment:
For later: I could see us potentially wanting to parameterize the position on screen (i.e. top right, bottom left, etc), in which case it might be good to pull these out into different classes, or deferring more control over to the lab on where to position it